tools/setup.ps1

```diff
@@ -562,15 +562,17 @@ function Install-Ebpf {
     $EbpfMsiFullPath = Get-EbpfMsiFullPath
     $EbpfMsiFullPath = (Resolve-Path $EbpfMsiFullPath).Path
 
+    Write-Verbose "Installing eBPF for Windows"
+
     if (Test-Path $EbpfPath) {
         Write-Error "$EbpfPath is already installed!"
     }
 
     # Try to install eBPF several times, since driver verifier's fault injection
     # may occasionally prevent the eBPF driver from loading.
     for ($i = 0; $i -lt 100; $i++) {
-        Write-Verbose "msiexec.exe /i $EbpfMsiFullPath INSTALLFOLDER=$EbpfPath ADDLOCAL=eBPF_Runtime_Components_JIT /qn /l*v $LogsDir\ebpfinstall.txt"
-        msiexec.exe /i $EbpfMsiFullPath INSTALLFOLDER=$EbpfPath ADDLOCAL=eBPF_Runtime_Components_JIT /qn /l*v $LogsDir\ebpfinstall.txt | Write-Verbose
+        Write-Verbose "msiexec.exe /i $EbpfMsiFullPath INSTALLFOLDER=$EbpfPath ADDLOCAL=eBPF_Runtime_Components,eBPF_Runtime_Components_JIT /qn /l*v $LogsDir\ebpfinstall.txt"
```

Comment:
If we're going to install a JIT variant, can we leave the .o files in the functional test? My preference would be to remove the JIT dependency uplevel (remove .o files) and install the official MSI on the newer systems, and install the special JIT-supporting build from the ZIP file on the downlevel tests. It seems inconsistent to remove .o files and also re-add JIT support.

Reply:
Fixed.